apps/www/_blog/2022-07-12-seen-by-in-postgresql.mdx

```diff
@@ -0,0 +1,736 @@
+
+# Wrap-up
+
+I hope you enjoyed this look down the trunk hole, and you've got an idea of how to implement solutions to surprisingly complex problems like this one with Postgres.
+
+As usual, Postgres has the tools to solve the problem _reasonably_ well (if not completely) before you reach out for more complicated/standalone solutions.
+
+**See any problems with the code, solutions that haven't been tried? -- reach out, or open an issue!**
+
+[pew-twitter-user-stats]: https://www.pewresearch.org/internet/2019/04/24/sizing-up-twitter-users/
+[k6]: https://k6.io
+[citus-columnar]: https://docs.citusdata.com/en/stable/admin_guide/table_management.html#columnar-storage
+[zedstore]: https://github.com/greenplum-db/postgres/tree/zedstore
+[pg-partman]: https://github.com/pgpartman/pg_partman
+[twitter-avg-tweets]: https://www.internetlivestats.com/twitter-statistics/
+[pg-docs-toast]: https://www.postgresql.org/docs/current/storage-toast.html
+[fujitsu-lz4-blog-post]: https://www.postgresql.fastware.com/blog/what-is-the-new-lz4-toast-compression-in-postgresql-14
+[kym-rest-of-owl]: https://knowyourmeme.com/memes/how-to-draw-an-owl
+[github-sraoss/pg_ivm]: https://github.com/sraoss/pg_ivm
+[age]: https://age.apache.org/
+[slonik-history]: https://www.vertabelo.com/blog/the-history-of-slonik-the-postgresql-elephant-logo/
+[citus]: https://www.citusdata.com/
+[autocannon]: https://www.npmjs.com/package/autocannon
+[pg-arrays]: https://www.postgresql.org/docs/current/arrays.html
+[pg-hstore]: https://www.postgresql.org/docs/current/hstore.html
+[pg-gist]: https://www.postgresql.org/docs/14/indexes-types.html#INDEXES-TYPE-GIST
+[pg-gin]: https://www.postgresql.org/docs/14/indexes-types.html#INDEXES-TYPES-GIN
+[wiki-muri]: https://en.wikipedia.org/wiki/Muri_(Japanese_term)
+[lz4]: https://github.com/lz4/lz4
+[clickhouse]: clickhouse.tech/
+[wiki-bloom-filter]: https://en.wikipedia.org/wiki/Bloom_filter
+[wiki-hll]: https://en.wikipedia.org/wiki/HyperLogLog
+[pg-cron]: https://github.com/citusdata/pg_cron
+[pg-fdw]: https://www.postgresql.org/docs/current/postgres-fdw.html
+[pg-view]: https://www.postgresql.org/docs/14/sql-createview.html
+[pg-tablespaces]: https://www.postgresql.org/docs/current/manage-ag-tablespaces.html
+[repo-scripts/generate/users.js]: https://gitlab.com/mrman/supabase-seen-by/-/blob/main/scripts/generate/users.js
+[repo-scripts/generate/posts.js]: https://gitlab.com/mrman/supabase-seen-by/-/blob/main/scripts/generate/posts.js
+[repo-scripts/bench.js]: https://gitlab.com/mrman/supabase-seen-by/-/blob/main/scripts/bench.js
+[repo-scripts/setup-request.cjs]: https://gitlab.com/mrman/supabase-seen-by/-/blob/main/scripts/setup-request.cjs
```

Comment:
Should we move all of these into a repo here? https://github.com/supabase-community/